tests/angular/package.json

```diff
@@ -1,20 +1,44 @@
 {
   "name": "@ni/angular-test",
   "version": "1.0.0",
-  "private": true,
   "description": "Smoke test for Angular project integration.",
-  "type": "module",
   "scripts": {
-    "test": "eslint ."
-  },
-  "peerDependencies": {
-    "@ni/eslint-config-angular": "*",
-    "@ni/eslint-config-javascript": "*"
+    "start": "ng serve",
+    "build": "ng build && ng build angular-library",
+    "lint": "ng lint",
+    "test": "ng lint && ng test",
+    "watch": "ng build --watch --configuration development"
   },
+  "private": true,
   "dependencies": {
-    "@angular/core": "^18.2.13"
+    "@angular/animations": "^18.2.0",
```

Comment:
Since `ng test` is running on CI it is now currently relying on whatever version of Chrome happens to be on the build agent. We generally would use [playwright](https://github.com/ni/nimble/blob/265570a90abc2dd3cacfcb84383cf0486150b8a4/packages/nimble-components/karma.conf.js#L11) or [puppeteer](https://github.com/karma-runner/karma-chrome-launcher?tab=readme-ov-file#headless-chromium-with-puppeteer) to bring in a versioned copy of Chrome for reproducibility so I'll leave the comment if you're up for it, but the repo is so small scoped / test suite so narrow that I don't feel particularly concerned about the build reproducibility here and am okay either way.

Reply:
I'm not going to worry about that right now. I'm already working on #176, and introducing that capability would derail my current direction quite a bit. Like yourself, I'm not concerned.